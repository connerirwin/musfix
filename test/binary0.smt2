; Distinguish between binary ops
(qualif Pos ((v Int)) (<= 0 v))
(qualif Neg ((v Int)) (<= v 0))

; Various Errors
; (qualif QQ ((b Bool)) (<= True b))
