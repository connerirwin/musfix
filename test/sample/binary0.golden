
Preparing to find greatest fixpoint...

Inputs
--------
Reading from file test/sample/binary0.msmt
Qualifier "Pos" [Var IntS "v"] (Binary Le (IntLit 0) (Var IntS "v"))
Qualifier "Neg" [Var IntS "v"] (Binary Le (Var IntS "v") (IntLit 0))

QMAP
--------
fromList []

Candidates
--------
[Candidate {solution = fromList [], validConstraints = fromList [], invalidConstraints = fromList [], label = "0"}]


Final candidates: 
0: [] (0 0)

